### PR TITLE
added Apple Watch App to store

### PR DESCRIPTION
**Pull request by tchristiansen**
I recently released an Apple Watch App to track kimai times anywhere. Here are my changes to list the app in the kimai.org marketplace.

### CODE REVIEW

**Comment by kevinpapst** [comment]
Hahaha, that is awesome ❤️ so there is finally a reason for me to buy an Apple Watch.
I would never have expected something like this, genius @tchristiansen !
And your beautiful landing page, just wow 👍 
Not entirely sure if there is a big market for this app, but I really like that it exists.

File: _store/umws-applewatch-app.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+**Note**:
+This app is not for offline tracking. It requires a connection to the API of your kimai server (v0.9 or higher - API v0.3).
+
+<p style="text-align:center"><a href="https://kimaiapp.umws.de">kimaiapp.umws.de</a></p>
+
+### App Screens
+
+<p style="text-align:center">Use your <strong>favourite</strong> or <strong>recent</strong> activities to start tracking immediately</p>
+<p style="text-align:center">
+    <img src="https://kimaiapp.umws.de//images/screens/kimaiTracker_favourites.png" width="200" style="margin:0 10px">
```
Comment (kevinpapst) [inline _store/umws-applewatch-app.md]: `kimaiapp.umws.de//images` vs `kimaiapp.umws.de/images`

Comment (tchristiansen) [reply _store/umws-applewatch-app.md]: Both should work, even if it was not my intention to add the double //

**Comment by tchristiansen** [comment]
> Hahaha, that is awesome ❤️ so there is finally a reason for me to buy an Apple Watch.

I was looking for a reason to keep mine.

**Comment by tchristiansen** [comment]
I changed the images to jpg (no alpha needed) to reduce the file size, and added them to the "images/marketplace/" folder.

**Pull request merged**